static/src/stylesheets/module/commercial/surveys/_survey.scss

```diff
@@ -42,9 +42,9 @@
         box-sizing: border-box;
 
         @include mq(tablet) {
-            left: 50%;
+            left: 42%;
```

Comment:
Is this supposed to be horizontally centered? Maybe you could keep the left:50% and just use `margin-left: -$gs-column-width * 5`, instead? That way it'd look a bit more like 'idiomatic' centering?